CHANGES.md

```diff
@@ -2,8 +2,24 @@
 
 ## not yet released
 
+## 2.3.0
+
+- `TOOLS-2372 need a tool to toggle branch protection rules for illumos-joyent`
+  This adds `jr github-settings -s <list of statuses> ...` to specify the
```

Comment:
s/jr github-settings/jr gh set-branch-protection/g

and in the two examples below as well.